### PR TITLE
[Ready] Wedding Content! Crate with tuxedo, wedding dress, cummerbund, veil, and craftable bouquets

### PR DESCRIPTION
**Pull request by OnlineGirlfriend**
## About The Pull Request

Adds a wedding crate which includes wedding dress, tuxedo, cummerbund (on the belt slot), veil, champagne, and three bouquets - mixed, sunflower, and poppy. The cost of the crate is 1500 credits. One each of wedding dress, tuxedo, cummerbund, and veil are available in the autodrobe.

Mixed bouquets = 2 lilies, 2 geraniums, 2 sunflowers
Sunflower bouquets = 6 sunflowers
Poppy bouquets = 6 poppies

I'd like to add a couple more bouquet options in the future.

I also moved Camera Obscura within recipes.dm to be with other misc. items.

## Why It's Good For The Game

Enhances wedding RP! More options for formalwear.

## Changelog
:cl:
add: wedding crate
add: tuxedo, wedding dress, veil, cummerbund, craftable bouquets
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
cummerbund

**Comment by TheFakeElon** [comment]
cute

**Comment by CRITAWAKETS** [comment]
homophobic content, should have two tuxedos and two wedding dresses in the crate in case you wish to perform a gay marriage

**Comment by scarwolff** [comment]
fuck this pr, empowering felinids like that, whats wrong with you.

**Comment by OnlineGirlfriend** [comment]
> homophobic content, should have two tuxedos and two wedding dresses in the crate in case you wish to perform a gay marriage

that is a good point

**Comment by tralezab** [comment]
Why not?

**Comment by Tlaltecuhtli** [comment]
coommerbund

**Comment by ATH1909** [comment]
It might be nice to allow any kinds of flowers to count for the bouquet, so that you don't need botany to grow 3 specific plants (or buy a shitton of wedding crates) to get a bunch of bouquets to hand out.

**Comment by GuyonBroadway** [comment]
Really this should go under the chapel supplies shouldn't it?

Hell give the chaplain a whole segment to do various ceremonies from weddings to that klingon pain ritual stuff.

**Comment by GuyonBroadway** [comment]
Also people are forgetting that weddings are like court cases on ss13.

A perfect antag oppertunity to show off your latest flamethrower mix.

File: code/modules/vending/autodrobe.dm [hunk 1]
```diff
@@ -140,7 +140,9 @@
 				   /obj/item/clothing/suit/chaplainsuit/clownpriest = 1,
 				   /obj/item/clothing/head/clownmitre = 1,
 				   /obj/item/skub = 1,
-				   /obj/item/clothing/suit/hooded/mysticrobe = 1)
+				   /obj/item/clothing/suit/hooded/mysticrobe = 1,
+				   /obj/item/clothing/under/dress/wedding_dress = 1,
+				   /obj/item/clothing/under/suit/tuxedo = 1)
```
Comment (Fikou) [inline code/modules/vending/autodrobe.dm:145]: they shouldnt be above skub...

Comment (Tlaltecuhtli) [reply code/modules/vending/autodrobe.dm:145]: undocumented and shouldnt be here, clowns dont need wedding dress

**Comment by Pugie** [comment]
wedding ring glove slot item would be cool or it could be an attachment to gloves.

**Comment by ExcessiveUseOfCobblestone** [comment]
can someone post the screenshot where the guy rationalizes his ERP by saying they're lawfully wedded, thanks.

File: code/modules/vending/autodrobe.dm (hunk 1 above)
Comment (OnlineGirlfriend) [reply code/modules/vending/autodrobe.dm:145]: it is in fact documented

**Comment by OnlineGirlfriend** [comment]
So thinking about the gay wedding thing, I opted to keep one of each outfit in the crate while lowering the cost so it's more reasonable to order two. The reason for this is that I want additional unique content in the crate, like the bouquets I just added and possibly rings or bridesmaid dresses.

**Comment by tmtmtl30** [comment]
>implying the best part of a gay wedding isn't the visual gag of who wears what

**Comment by OnlineGirlfriend** [comment]
pardon me while I learn how to use a computer

**Comment by TheFakeElon** [comment]
Port rings from austation 😳

**Review by Twaticus — CHANGES_REQUESTED** [review]
the tuxedo in icons/obj/clothing/under/suits.dmi is using directionals when it should just be 1 sprite and it needs be cropped to match the other suits in that file.
the cumbund thing in icons/mob/clothing/belt.dmi is missing a name

**Comment by OnlineGirlfriend** [comment]
I will update this tonight or tomorrow, thanks for the feedback!

Also if someone could remove UI label, this doesn't touch UI.

**Review by Twaticus — APPROVED** [review]

**Comment by Tumesuo** [comment]
lol cum

**Pull request merged**